dp_consistency/tests/tools/test_correctoriginreport.py

```diff
@@ -10,8 +10,7 @@
 MRC_FILTER = "MRC"
 
 
-class TestReport(TestCase):
-
+class TestOriginSheetReport(TestCase):
```

Comment:
did you include an updated test for N EDPs?

Reply:
Not doing this per offline convo, will tackle this for the report proto